hoomd/md/integrate.py

```diff
@@ -93,6 +100,13 @@ def _children(self):
 
         return children
 
+    def _setattr_param(self, attr, value):
+        if attr == "rigid":
+            if self._attached:
+                self._rigid._attach()
+                self._cpp_obj.rigid = self._rigid._cpp_obj
+        else:
+            super()._setattr_param("rigid", None)
```

Comment:
```suggestion
            super()._setattr_param(attr, value)
```
Maybe? @b-butler what should this be?